src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SoftwareChoicesStepDef.scala

```diff
@@ -129,6 +132,9 @@ class SoftwareChoicesStepDef extends BaseStepDef {
     driver
       .findElement(By.cssSelector(".apply-filters-button"))
       .click()
+
+    // service reacts quicker than tests to accurately wait for loading icons
+    Thread.sleep(100)
```

Comment:
`sleep` can be replaced with the following code:
```suggestion
    fluentWait
      .until(
        ExpectedConditions.visibilityOfElementLocated(By.id(softwareVendorsId))
      )
```